content/applications/sales/point_of_sale/payment_methods/terminals/viva_wallet.rst

```diff
@@ -0,0 +1,159 @@
+===========
+Viva Wallet
+===========
+
+Connecting a **Viva Wallet payment terminal**  allows you to offer a fluid payment flow to your
+customers and ease the work of your cashiers.
+
+.. note::
+   Viva Wallet lets you turn your phone into a mobile card reader: `Tap On Phone
+   <https://www.vivawallet.com/gb_en/blog-tap-on-phone-gb>`_.
+
+Configuration
+=============
+
+Start by creating your Viva Wallet account on `Viva Wallet's website <https://www.vivawallet.com>`_.
+
+Locate your Viva Wallet credentials
+-----------------------------------
+
+When configuring Viva Wallet in Point of Sale, you need to use specific credentials that are
+available in your Viva Wallet account. These credentials include your :ref:`Merchant ID
+<viva_wallet/id-key>`, :ref:`API key <viva_wallet/id-key>`, :ref:`POS APIs credentials
+<viva_wallet/pos-apis>`, and :ref:`Terminal ID <viva_wallet/identifier>` number.
+
+.. _viva_wallet/id-key:
+
+Merchant ID and API key
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To find your Merchant ID and API key:
+
+#. Go to your Viva Wallet account and select the relevant account;
+#. Go to :menuselection:`Settings --> API access` in the navigation menu.
+
+Your Merchant ID and API Key are located in the :guilabel:`Access credentials` section at the top of
+the page. Save the keys and paste them into the Odoo :guilabel:`Merchant ID` and :guilabel:`API Key`
+fields :ref:`when creating a payment method <viva_wallet/method-creation>`.
```

Comment:
This link (:ref:`when creating a payment method <viva_wallet/method-creation>`) redirects to another section of the page, that redirects to another page (when you click on "in the application settings"), that redirects to "Access POS settings" page. I am lost and I don't know where I have to paste the credentials.

Reply:
I tried to simplified the paragraph, I hope it helps